lte/gateway/c/oai/tasks/amf/amf_recv.cpp

```diff
@@ -118,6 +118,7 @@ int amf_handle_registration_request(
     OAILOG_DEBUG(LOG_NAS_AMF, "Processing REGITRATION_REQUEST message\n");
   }
   amf_registration_request_ies_t* params = new (amf_registration_request_ies_t);
```

Comment:
when is this dynamically allocated memory gets freed up?

Reply:
This should be freed when UE moved to DEREGISTERED

Reply:
once procedure is completed and working on to resolve this